openvpn-server/conf/server.conf

```diff
@@ -10,7 +10,8 @@ key /etc/openvpn/secrets/server.key  # This file should be kept secret
 
 tls-version-min 1.3 or-highest
 tls-cipher TLS-ECDHE-RSA-WITH-AES-256-GCM-SHA384
-ncp-ciphers AES-256-GCM
+data-ciphers AES-256-GCM
+data-ciphers-fallback -
```

Comment:
Our clients are currently OpenVPN 2.4. My understanding is that this fallback is needed for these clients (["Openvpn Clients which do not support --data-ciphers (Openvpn 2.4, 2.3)"](https://community.openvpn.net/openvpn/wiki/CipherNegotiation#Effectivedirectivesandterms)). When setting it to `-`, I would think that no connection could be established.

It seems to me that the correct setting would be to set the fallback to one of the algorithms in `data-ciphers`, of which we have only one, so `data-ciphers-fallback AES-256-GCM` would be correct. Do you think that makes sense?

Reply:
The documentation is a bit confusing. The cipher example does not mention `data-ciphers-fallback` but shows a valid connection 😕 : https://community.openvpn.net/openvpn/wiki/CipherNegotiation#Serverversion2.5
Anyway I think the fallback will not harm the security.